src/components/Card/src/CardHeader/CardHeader.tsx

```diff
@@ -54,7 +58,12 @@ export interface CardHeaderProps extends BaseProps {
  * Primary UI component for user interaction
  */
 export const CardHeader: React.FC<CardHeaderProps> = (props: CardHeaderProps) => {
-  return <MaterialCardHeader {...props}>{props.children}</MaterialCardHeader>;
+  return (
+    <StylesProvider injectFirst>
+      <MaterialCardHeader {...props}>{props.children}</MaterialCardHeader>
```

Comment:
`{...props}`

Reply:
Cardheader has been removed since we don't actually use it in card